core/src/main/java/com/chopshop166/chopshoplib/motors/CSSparkMax.java

```diff
@@ -11,16 +13,51 @@
  */
 public class CSSparkMax extends CSSpark {
 
+    /**
+     * Create a Spark MAX associated with a brushed motor.
+     * 
+     * @param deviceID The CAN device ID
+     * @param countsPerRev The counts per revolution of the encoder.
+     * @return A CSSparkMax object.
+     */
+    public static CSSparkMax brushed(final int deviceID, final int countsPerRev) {
+        final var spark = new CANSparkMax(deviceID, MotorType.kBrushed);
+        final var enc = spark.getEncoder(SparkRelativeEncoder.Type.kQuadrature, countsPerRev);
+        return new CSSparkMax(spark, enc);
+    }
+
+    /**
+     * Create a Spark MAX associated with a Neo or Neo 550 motor.
+     * 
+     * @param deviceID The CAN device ID
+     * @return A CSSparkMax object.
+     */
+    public static CSSparkMax neo(final int deviceID) {
+        final var spark = new CANSparkMax(deviceID, MotorType.kBrushed);
+        final var enc = spark.getEncoder(SparkRelativeEncoder.Type.kHallSensor, 42);
+        return new CSSparkMax(spark, enc);
+    }
+
+    /**
+     * Create a Spark MAX associated with a Neo Vortex motor.
+     * 
+     * @param deviceID The CAN device ID
+     * @return A CSSparkMax object.
+     */
+    public static CSSparkMax vortex(final int deviceID) {
+        final var spark = new CANSparkMax(deviceID, MotorType.kBrushed);
+        final var enc = spark.getEncoder(SparkRelativeEncoder.Type.kQuadrature, 7168);
+        return new CSSparkMax(spark, enc);
+    }
+
     /**
      * Create a new wrapped SPARK MAX Controller.
      *
-     * @param deviceID The device ID.
-     * @param type The motor type connected to the controller. Brushless motors must be connected to
-     *        their matching color and the hall sensor plugged in. Brushed motors must be connected
-     *        to the Red and Black terminals only.
+     * @param spark The spark object to wrap.
+     * @param enc The encoder object.
      */
-    public CSSparkMax(final int deviceID, final MotorType type) {
-        super(new CANSparkMax(deviceID, type), type);
+    public CSSparkMax(final CANSparkMax spark, final RelativeEncoder enc) {
```

Comment:
Could we have another constructor that takes the deviceID only and configures the SparkMax for a NEO? This is the default use-case for a `CSSparkMax` so it would be nice to make that easy.

Reply:
Not the way Java works